napalm/eos/eos.py

```diff
@@ -1699,7 +1712,7 @@ def _parse_per_peer_bgp_detail(peer_output):
                     True if item["local_address"] else False
                 )
                 item["multihop"] = (
-                    False if item["multihop"] == 0 or item["multihop"] == "" else True
+                    False if item["multihop"] == "0" or item["multihop"] == "" else True
```

Comment:
In the TexFSM template, the value would seem to be extracted as a number (`Value multihop (\d+)`). Could you clarify why comparing against string (`"0"`) here?

Reply:
Yes, so the TexFSM template is expecting a regular expression (\d+) for the multihop variable but the output remains a string : [example](https://pyneng.readthedocs.io/en/latest/book/21_textfsm/README.html#:~:text=The%20result%20of%20script%20execution)
that is also why the eos.py file still uses the "napalm.base.helpers.convert()" function to convert some of the extracted variables to int.